### PR TITLE
Pme human alignment

### PR DESCRIPTION
**Pull request by grgrzybek**

### CODE REVIEW

File: modules/fuse-credential-store/fuse-credential-store-commands/src/main/java/org/jboss/fuse/credential/store/command/CreateCredentialStore.java [hunk 1]
```diff
@@ -136,6 +136,7 @@ public Object execute() throws Exception {
                 System.err.println("Passwords do not match.");
                 return null;
             }
+            password = password1;
```
Comment (orpiske) [inline modules/fuse-credential-store/fuse-credential-store-commands/src/main/java/org/jboss/fuse/credential/store/command/CreateCredentialStore.java:139]: Is this related to the PME change?

Comment (grgrzybek) [reply modules/fuse-credential-store/fuse-credential-store-commands/src/main/java/org/jboss/fuse/credential/store/command/CreateCredentialStore.java:139]: fortunately no ;) My mistake.

Comment (grgrzybek) [reply modules/fuse-credential-store/fuse-credential-store-commands/src/main/java/org/jboss/fuse/credential/store/command/CreateCredentialStore.java:139]: I didn't fully test non-batch scenario. Or rather didn't check this part when adding batch support.

**Review by orpiske — APPROVED** [review]
LGTM.
